src/wp-includes/fonts/font-library/class-wp-font-library.php

```diff
@@ -0,0 +1,131 @@
+<?php
+/**
+ * Font Library class.
+ *
+ * This file contains the Font Library class definition.
+ *
+ * @package    WordPress
+ * @subpackage Font Library
+ * @since      6.4.0
+ */
+
+/**
+ * Font Library class.
+ *
+ * @since 6.4.0
+ */
+class WP_Font_Library {
+
+	const PHP_7_TTF_MIME_TYPE = PHP_VERSION_ID >= 70300 ? 'application/font-sfnt' : 'application/x-font-ttf';
+
+	const ALLOWED_FONT_MIME_TYPES = array(
+		'otf'   => 'font/otf',
+		'ttf'   => PHP_VERSION_ID >= 70400 ? 'font/sfnt' : self::PHP_7_TTF_MIME_TYPE,
+		'woff'  => PHP_VERSION_ID >= 80100 ? 'font/woff' : 'application/font-woff',
+		'woff2' => PHP_VERSION_ID >= 80100 ? 'font/woff2' : 'application/font-woff2',
+	);
```

Comment:
i don't love this conditional logic here. How about property that is set in the __constructor with the conditional logic. Also a comment would be handy here explain this logic.

Reply:
Nope, it's not ideal, I think a fix in core to support multiple mime types for each file extension is needed. I'll add a comment explaining the logic. I think we can work on the core fix as a follow-up.

Reply:
Following up:

This was refactored to remove the constant in favor of a static method called `get_font_mime_types()`. As `WP_Font_Library` is a static class design pattern, there is no constructor to initialize a property. The new `get_font_mime_types()()` method has the conditional logic to build and return the mime type for the running PHP version.